flan/src/lib/FileInput.svelte

```diff
@@ -46,6 +45,31 @@
 						Remove image
 					</button>
 				</div>
+			{:else if inputFile.type.includes('audio/')}
+				<div
+					class="grid grid-rows-[1fr_auto] gap-4 bg-secondary text-white p-4 rounded-lg">
+					<!-- class="grid grid-rows-[1fr_auto] gap-4" -->
+					<code class="break-all">{inputFile.name}</code>
+					<audio controls class="w-full h-12" src={URL.createObjectURL(inputFile)} />
+					<button
+						class="btn variant-filled-secondary z-10 rounded-lg"
```

Comment:
theres no discernable difference in remove file buttons, placing outside if scope might be best

Reply:
addressing this as well